server/pulp/server/db/manage.py

```diff
@@ -200,6 +210,9 @@ def main():
         return os.EX_DATAERR
     except models.MigrationRemovedError:
         return os.EX_SOFTWARE
+    except ServerSelectionTimeoutError:
```

Comment:
If the mongodb service is down the exception is raised when _CONNECTION.server_info() is called at:
https://github.com/pulp/pulp/blob/master/server/pulp/server/db/connection.py#L119

currently we're not catching this exception and show a stack trace to the user